packages/cli/src/auth.ts

```diff
@@ -35,21 +34,27 @@ whoami.action(async (options) => {
   printMe(medplum);
 });
 
-async function startLogin(medplum: MedplumClient, profile: Profile): Promise<void> {
+async function startLogin(medplum: MedplumClient, profile?: Profile): Promise<void> {
```

Comment:
What is the case where `profile` is undefined?

Reply:
running `medplum login` will have profile as undefined

Reply:
Good point, it's never